### PR TITLE
Add package version metadata

### PR DESCRIPTION
**Pull request by darkfeline**
Needed for a future commit to add package.el support.

### CODE REVIEW

**Review by phst — APPROVED** [review]

File: elisp/proto/generate.el [hunk 1]
```diff
@@ -14,6 +14,8 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 
+;; Version: 0.1.0
```
[the remark this answers is not in the document]
Comment (darkfeline) [reply elisp/proto/generate.el:17]: tl;dr 0 versions are not valid.  As you probably know, these get converted to lists of numbers, and the zero version is used as a "null" value in various places (and e.g. `(0)` and `(0 0 0)` are equal)

Beside that, the only thing special is that it's the starting version under SemVer.

[the remark this answers is not in the document]
Comment (phst) [reply elisp/proto/generate.el:17]: SG. Please make the unit tests pass (probably need to fix the line numbers in the coverage test file).

**Comment by darkfeline** [comment]
Running `bazel run -- //tests:go_default_test --regenerate-coverage-dat` seems to hang for me after 

```
INFO: Running command line: external/bazel_tools/tools/test/test-setup.sh tests/go_default_test_/go_default_test '--binary=_main/tests/test_test' --regenerate-coverage-dat
exec ${PAGER:-/usr/bin/less} "$0" || exit 1
Executing tests from //tests:go_default_test
-----------------------------------------------------------------------------
```

Can you reproduce?

**Comment by phst** [comment]
> Running `bazel run -- //tests:go_default_test --regenerate-coverage-dat` seems to hang for me after
> 
> ```
> INFO: Running command line: external/bazel_tools/tools/test/test-setup.sh tests/go_default_test_/go_default_test '--binary=_main/tests/test_test' --regenerate-coverage-dat
> exec ${PAGER:-/usr/bin/less} "$0" || exit 1
> Executing tests from //tests:go_default_test
> -----------------------------------------------------------------------------
> ```
> 
> Can you reproduce?

Works as expected for me

**Comment by darkfeline** [comment]
I think it's fixed now.  For some reason bazel for this repo doesn't work right on gLinux and I can't be bothered to debug it; it works on a plain Debian box.

**Pull request merged**